client-csharp/SessionPool.cs

```diff
@@ -0,0 +1,642 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+using Thrift;
+using Thrift.Transport;
+using Thrift.Protocol;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using Thrift.Transport.Client;
+using iotdb_client_csharp.client.utils;
+using NLog;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+namespace iotdb_client_csharp.client{
+    public class SessionPool{
+        private string username, password, zoneId, host;
+        public int SUCCESS_CODE{
```

Comment:
can we use [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-5.0) Enum instead of int type?

Reply:
Sounds Good